src/app/shared/components/history-log-table/history-log-table.component.html

```diff
@@ -0,0 +1,119 @@
+<div class="wrapper">
+  <div class="table-container mat-elevation-z8">
```

Comment:
I think this component should be moved to the data-module, to the history-log folder, because this view is to specific, don't think we can reuse it somewhere